assets/js/components/wp-dashboard/WPDashboardApp.js

```diff
@@ -84,6 +85,7 @@ const WPDashboardApp = () => {
 							) }
 						</div>
 					) }
+					<WPDashboardPopularPages />
```

Comment:
This must only be rendered if the Analytics Module is active and connected, similar to the other Analytics components above - otherwise it will cause breakage when the module is not active/not connected.

Reply:
Ah, thank you. I did wonder, but I couldn't see anything specific in the issue about this. I'll add it in 👍